e2e/playwright/support/utils.ts

```diff
@@ -0,0 +1,66 @@
+import path from 'node:path';
+
+import type {Page} from '@playwright/test';
+
+import {ChannelsPage} from '@e2e-support/ui/pages';
+import {UserProfile} from '@mattermost/types/users';
+import Client4 from '@mattermost/client/client4';
+
+const SCREENSHOTS_DIR = path.join(__dirname, '../screenshots');
+
+export const sleep = (page: Page, millis = 500) => page.waitForTimeout(millis);
```

Comment:
0/5 if this still provides value

Reply:
I'd be more inclined to use the native directive in the real tests, so we don't create an intermediate language